docs/style-guide.rst

```diff
@@ -888,13 +888,19 @@ Function Names
 Functions other than constructors should use mixedCase. Examples: ``getBalance``, ``transfer``, ``verifyOwner``, ``addMember``, ``changeOwner``.
 
 
+Internal Function Names
+=======================
+
+Internal functions which have the ``internal`` or ``private`` modifiers should use mixedCase with leading underscore. Examples: ``_processTransaction``, ``_addMember``, ``_modifyConfig``.
+
+
 Function Argument Names
 =======================
 
-Function arguments should use mixedCase. Examples: ``initialSupply``, ``account``, ``recipientAddress``, ``senderAddress``, ``newOwner``.
+Function arguments should use mixedCase with leading underscore. Examples: ``_initialSupply``, ``_account``, ``_recipientAddress``, ``_senderAddress``, ``_newOwner``.
```

Comment:
I think I agree with this, it is also a convention in C++ and has the immediate benefits when assigning to storage variables (no clash between the variable names).

Reply:
Sorry, but I strongly disagree. While this is nice for the code of the function, this also has an influence on automatically generated interfaces to the contracts, where it will look very weird.